version.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module DynatraceConstants
```

Comment:
Should we leave this at top-level or move it into `lib/logstash/outputs/`?

Reply:
If we move it into `lib` the CI fails. The only files added into the CI are `Gemfile *.gemspec VERSION* version*` when the CI runs `bundle install`. It doesn't copy the rest of the plugin into the image until after that. This is to improve caching for their builds. Changes to lib will be very frequent but changes to dependencies are rare, so dependencies should be cached in a layer that doesn't break when lib is updated. This saves the docker cache from doing the expensive `bundle install` every time lib is updated.

Reply:
ref: https://github.com/logstash-plugins/.ci/blob/main/Dockerfile#L19